### ISSUE TITLE
EventBus default LoggingSubscriberExceptionHandler does not log stacktrace correctly

### ISSUE THREAD

**Issue by Milbor** [google/guava#2220]
See EventBus.LoggingSubscriberExceptionHandler.handleException method:
```java
    @Override
    public void handleException(Throwable exception,
        SubscriberExceptionContext context) {
      logger.log(Level.SEVERE, "Could not dispatch event: " 
          + context.getSubscriber() + " to " + context.getSubscriberMethod(),
          exception.getCause()/*here I would expect just exception, not the cause*/);
    }
```
If exception is thrown in subscriber method, only stacktrace of exception cause is logged. If this exception does not have any cause, no stacktrace is logged.
Argument *exception* of this method is already exception thrown from the subscriber method, not any wrapped exception.

**Comment by Quurks** [comment]
Seems to be a duplicate of #2093

**Comment by Milbor** [comment]
Yes it is, so it will be fixed in 19.0. Thank you.
